e2e/cypress/integration/system_console/environment_spec.js

```diff
@@ -0,0 +1,331 @@
+
+            // # Close the modal
+            cy.get('#teamSettingsModalLabel').find('button').should('be.visible').click();
+        });
+
+        // Validate that the image is being displayed
+        cy.get(`#${testTeam.name}TeamButton`).scrollIntoView().within(() => {
+            cy.findByTestId('teamIconImage').then((imageDiv) => {
+                const url = imageDiv.css('background-image').split('"')[1];
+
+                // # Verify that the image is the correct one
+                cy.fixture(mattermostIcon).then((overrideImage) => {
+                    cy.request({url, encoding: 'base64'}).then((response) => {
+                        expect(response.status).to.equal(200);
+                        expect(response.body).to.eq(overrideImage);
+                    });
+                });
+            });
+        });
+    });
+
+    it('MM-T991 - Database fields can be edited and saved', () => {
+        cy.visit('/admin_console/environment/database');
+
+        const queryTimeoutValue = 100;
+        const maxOpenConnsValue = 1000;
+        cy.findByTestId('queryTimeoutinput').clear().type(queryTimeoutValue);
+        cy.findByTestId('maxOpenConnsinput').clear().type(maxOpenConnsValue);
+
+        // # Click Save button to save the settings
+        cy.get('#saveSetting').click();
+
+        // Get config again
+        cy.apiGetConfig().then(({config}) => {
+            // * Verify the database setting values are saved
+            expect(config.SqlSettings.QueryTimeout).to.eq(queryTimeoutValue);
+            expect(config.SqlSettings.MaxOpenConns).to.eq(maxOpenConnsValue);
+        });
+    });
+
+    it('MM-T993 - Minimum hashtag length at least 2', () => {
+        cy.visit('/admin_console/environment/database');
+
+        const minimumHashtagOrig = 3;
+        const minimumHashtagLength1 = 2;
+        const minimumHashtagLength2 = 1;
+
+        cy.findByTestId('minimumHashtagLengthinput').clear().type(minimumHashtagLength1);
+
+        // # Click Save button to save the settings
+        cy.get('#saveSetting').click();
+
+        // Get config again
+        cy.apiGetConfig().then(({config}) => {
+            // * Verify the database setting value is saved
+            expect(config.ServiceSettings.MinimumHashtagLength).to.eq(minimumHashtagLength1);
+        });
+
+        cy.findByTestId('minimumHashtagLengthinput').clear().type(minimumHashtagLength2);
+
+        // # Click Save button to save the settings
+        cy.get('#saveSetting').click();
+
+        // Get config again
+        cy.apiGetConfig().then(({config}) => {
+            // * Verify the database setting value is reset to original value
+            expect(config.ServiceSettings.MinimumHashtagLength).to.eq(minimumHashtagOrig);
+        });
+    });
+
+    it('MM-T994 - Fields editable when enabled, but not saveable until validated', () => {
+        // * Check if server has license for Elasticsearch
+        cy.apiRequireLicenseForFeature('Elasticsearch');
```

Comment:
We're separating EE only from other tests. With that, please move this test into its own spec file at `e2e/cypress/integration/enterprise/system_console` folder. Can be of the same filename.